locales/de.json

```diff
@@ -1,574 +1,911 @@
-			"pass": "",
-			"fail": "Das Element besitzt ein value-Attribut und das value-Attribut ist leer."
-		},
-		"non-empty-title": {
-			"pass": "",
-			"fail": "Das Element besitzt kein title-Attribut oder das title-Attribut ist leer."
-		},
-		"non-empty-value": {
-			"pass": "",
-			"fail": "Das Element besitzt kein value-Attribut oder das value-Attribut ist leer."
-		},
-		"role-none": {
-			"pass": "",
-			"fail": "Die Standard-Semantik des Elementes ist nicht mit role=\"none\" überschrieben."
-		},
-		"role-presentation": {
-			"pass": "",
-			"fail": "Die Standard-Semantik des Elementes ist nicht mit role=\"presentation\" überschrieben."
-		},
-		"caption-faked": {
-			"pass": "",
-			"fail": "Die erste Zeile der Tabelle sollte eine Tabellenüberschrift anstelle einer Tabellenzelle sein."
-		},
-		"html5-scope": {
-			"pass": "",
-			"fail": "In HTML5 dürfen scope-Attribute lediglich für Tabellenkopfzellen <th> verwendet werden."
-		},
-		"same-caption-summary": {
-			"pass": "",
-			"fail": "Die Inhalte des summary-Attributes und des <caption>-Elementes sind identisch."
-		},
-		"scope-value": {
-			"pass": "",
-			"fail": "Das <td>-Element besitzt ein scope-Attribut. In HTML5 dürfen scope-Attribute jedoch lediglich für Tabellenkopfzellen <th> verwendet werden."
-		},
-		"td-has-header": {
-			"pass": "",
-			"fail": "Nicht alle (nicht-leeren) Datenzellen haben eine Tabellenkopfzelle."
-		},
-		"td-headers-attr": {
-			"pass": "",
-			"fail": "Das headers-Attribut wird nicht ausschließlich dafür verwendet, um auf andere Zellen in der Tabelle zu verweisen."
-		},
-		"th-has-data-cells": {
-			"pass": "",
-			"fail": "Nicht alle Tabellenkopfzellen beziehen sich auf Datenzellen. ",
-			"incomplete": "Datenzellen der Tabelle fehlen oder sind leer."
-		},
-		"hidden-content": {
-			"pass": "",
-			"fail": "Beim Analysieren der Inhalte auf dieser Seite sind Probleme aufgetreten.",
-			"incomplete": "Auf der Seite befinden sich versteckte Inhalte, die nicht analysiert werden konnten. Um den Inhalt analysieren zu können, müssen Sie die Anzeige auslösen."
-		}
-	},
-	"failureSummaries": {
-		"any": {
-			"failureMessage": "Korrigiere mindestens einen der folgenden Punkte:{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
-		},
-		"none": {
-			"failureMessage": "Korrigiere alle der folgenden Punkte:{{~it:value}}\n  {{=value.split('\\n').join('\\n  ')}}{{~}}"
-		}
-	},
-	"incompleteFallbackMessage": ""
+  "lang": "de",
+  "rules": {
+    "accesskeys": {
+      "description": "",
+      "help": "Der Wert des accesskey-Attributes muss einzigartig sein."
+    },
+    "area-alt": {
+      "description": "",
+      "help": "Aktive <area>-Elemente müssen einen Alternativtext besitzen."
+    },
+    "aria-allowed-attr": {
+      "description": "",
+      "help": "Elemente dürfen nur erlaubte ARIA-Attribute verwenden."
+    },
+    "aria-allowed-role": {
+      "description": "Der Wert des role-Attributes muss gültig sein.",
```

Comment:
description and help need to correspond, these are slightly different. The hep text is accurate please change the description to match.